### ISSUE TITLE
Undefined offset on feed

### ISSUE THREAD

**Issue by Mte90** [gocodebox/lifterlms#408]
### Error Messages / Logs

```[16-Jan-2018 17:29:08 UTC] Array
(
    [error] => Notice
    [file] => /home/domain/wp-content/plugins/lifterlms/vendor/ericmann/wp-session-manager/includes/class-wp-session.php
    [line] => 77
    [context] => Array
        (
            [cookie] => cxxxxxxxxxxx
            [cookie_crumbs] => Array
                (
                    [0] => cxxxxxxxxxxx
                )

        )

    [url] => https://domain/author/andrea/feed/
    [filter] => plugins_loaded
    [path] => /home/domain/wp-content/plugins/lifterlms/vendor/ericmann/wp-session-manager/includes/class-wp-session.php
    [message] => Notice (8): Undefined offset: 1 in [/home/domain/wp-content/plugins/lifterlms/vendor/ericmann/wp-session-manager/includes/class-wp-session.php, line 77]
)

[16-Jan-2018 17:29:08 UTC] Array
(
    [error] => Notice
    [file] => /home/domain/wp-content/plugins/lifterlms/vendor/ericmann/wp-session-manager/includes/class-wp-session.php
    [line] => 78
    [context] => Array
        (
            [cookie] => cxxxxxxxxxxx
            [cookie_crumbs] => Array
                (
                    [0] => cxxxxxxxxxxx
                )

        )

    [url] => https://domain/author/andrea/feed/
    [filter] => plugins_loaded
    [path] => /home/domain/wp-content/plugins/lifterlms/vendor/ericmann/wp-session-manager/includes/class-wp-session.php
    [message] => Notice (8): Undefined offset: 2 in [/home/domain/wp-content/plugins/lifterlms/vendor/ericmann/wp-session-manager/includes/class-wp-session.php, line 78]
)```

I found this on my log.

**Comment by thomasplevy** [comment]
Thanks for the heads up and sorry for this issue. This is stemming from a library I'm working on getting rid of entirely so while notices in logs are important, I'd need to get this fixed in the library and since I'm planning to get rid of the library all together I'm going to let this sit for the moment as relatively low priority.

My plan is to have this lib removed in association with updates planned around #371 which should be released mid-late feb with any luck

**Comment by thomasplevy** [comment]
Closed in Version 4.0.0
